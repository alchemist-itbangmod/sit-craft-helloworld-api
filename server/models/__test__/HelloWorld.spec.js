const HelloWorld = require('../HelloWorld.model')

describe('HelloWorldQuery', ()=> {
  it(`Query 'hello-world' Correctly!`, async () => {
    try {
      const helloworld = new HelloWorld()
      await helloworld.getHelloWorld()

      expect(prepareMongooseObject(helloworld.query.hello_world)).toMatchSnapshot()
    } catch (error) {
      expect(error).toMatchSnapshot()
    }
  })
})