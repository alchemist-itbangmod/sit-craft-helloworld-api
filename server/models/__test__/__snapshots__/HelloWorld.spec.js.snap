// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorldQuery Query 'hello-world' Correctly! 1`] = `[TypeError: HelloWorld is not a constructor]`;
